apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -591,7 +592,9 @@ NetSimPacketEditor.prototype.bindElements_ = function () {
     }
   ];
 
-  rowTypes.forEach(function (rowType) {
+  rowTypes.filter(function (rowType) {
+    return this.isEncodingEnabled_(rowType.typeName);
+  }, this).forEach(function (rowType) {
```

Comment:
This feels like a good candidate for a helper method.

```javascript
NetSimPacketEditor.prototype.getEnabledRowTypes_ = function () {
  return rowTypes.filter(function (rowType) {
    return this.isEncodingEnabled_(rowType.typeName);
  }, this);
};
// ...
this.getEnabledRowTypes_.forEach(function (rowType) {
  // ...
```
I'm also realizing here that you'd need the rowTypes template we currently build up every time we call this function should maybe live cached somewhere else.  Might be a good change anyway.